stable/chaoskube/Chart.yaml

```diff
@@ -15,4 +15,6 @@ maintainers:
   email: linki+kubernetes.io@posteo.de
 - name: alexvicegrab
   email: a.vicente.grab@gmail.com
-engine: gotpl
+- name: syedimam0012
+  email: syedimam0012@gmail.com
+engine: gotpl
```

Comment:
@syedimam0012 There needs to be a newline here to make the linter happy:

```console
stable/chaoskube/Chart.yaml
  20:14     error    no new line character at the end of file  (new-line-at-end-of-file)
```

Reply:
Looks good now. Cheers